scss/_patterns_navigation.scss

```
@@ -282,7 +282,7 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
     .p-navigation__link {
       @extend %navigation-link;
 
-      padding-left: $sph--x-large;
+      padding-left: calc($sph--x-large + 0.19rem);
```

In case adding such magic numbers we need a comment that clarifies why they are there
```suggestion
      padding-left: calc($sph--x-large + 0.19rem); // additional padding added to align better with logos as originally designed in SVG
```